### ISSUE TITLE
test_cache_last_used_update test fails

### ISSUE THREAD

**Issue by DerDakon** [davidhalter/parso#193]
When running the tests of 0.8.2 on Gentoo Linux with Python 3.9.6 on a HPPA C8000 (1GHz):

```
_________ test_cache_last_used_update[False-False] _________

diff_cache = False, use_file_io = False

    @pytest.mark.parametrize('diff_cache', [False, True])
    @pytest.mark.parametrize('use_file_io', [False, True])
    def test_cache_last_used_update(diff_cache, use_file_io):
        p = Path('/path/last-used')
        parser_cache.clear()  # Clear, because then it's easier to find stuff.
        parse('somecode', cache=True, path=p)
        node_cache_item = next(iter(parser_cache.values()))[p]
        now = time.time()
>       assert node_cache_item.last_used < now
E       assert 1628015776.292685 < 1628015776.292685
E        +  where 1628015776.292685 = <parso.cache._NodeCacheItem object at 0xf68062b0>.last_used

diff_cache = False
node_cache_item = <parso.cache._NodeCacheItem object at 0xf68062b0>
now        = 1628015776.292685
p          = PosixPath('/path/last-used')
use_file_io = False

test/test_cache.py:140: AssertionError
__________ test_cache_last_used_update[True-True] __________

diff_cache = True, use_file_io = True

    @pytest.mark.parametrize('diff_cache', [False, True])
    @pytest.mark.parametrize('use_file_io', [False, True])
    def test_cache_last_used_update(diff_cache, use_file_io):
        p = Path('/path/last-used')
        parser_cache.clear()  # Clear, because then it's easier to find stuff.
        parse('somecode', cache=True, path=p)
        node_cache_item = next(iter(parser_cache.values()))[p]
        now = time.time()
>       assert node_cache_item.last_used < now
E       assert 1628015777.020685 < 1628015777.020685
E        +  where 1628015777.020685 = <parso.cache._NodeCacheItem object at 0xf6a95dc0>.last_used

diff_cache = True
node_cache_item = <parso.cache._NodeCacheItem object at 0xf6a95dc0>
now        = 1628015777.020685
p          = PosixPath('/path/last-used')
use_file_io = True

test/test_cache.py:140: AssertionError
====================== short test summary info =======================
FAILED test/test_cache.py::test_cache_last_used_update[False-False] - assert 1628015776.292685 < 1628015776.292685
FAILED test/test_cache.py::test_cache_last_used_update[True-True] - assert 1628015777.020685 < 1628015777.020685
```

With 0.7.1 it is similar:

```
FAILED test/test_cache.py::test_cache_last_used_update[False-False] - assert 1628016547.492685 < 1628016547.492685
FAILED test/test_cache.py::test_cache_last_used_update[False-True] - assert 1628016548.2086852 < 1628016548.2086852
FAILED test/test_cache.py::test_cache_last_used_update[True-False] - assert 1628016548.2646852 < 1628016548.2646852
FAILED test/test_cache.py::test_cache_last_used_update[True-True] - assert 1628016548.316685 < 1628016548.316685
```

**Comment by davidhalter** [comment]
This was fixed in 60fed7b9 (not part of a release, yet).

**Comment by mgorny** [comment]
Thanks, the fix looks reasonable ;-).
